src/libmongoc/tests/test-mongoc-structured-log.c

```diff
@@ -699,6 +738,74 @@ test_structured_log_component_names (void)
    ASSERT_CMPSTR (mongoc_structured_log_get_component_name (component), "connection");
 }
 
+void
+test_structured_log_max_document_length (void)
+{
+   mongoc_structured_log_opts_t *opts = mongoc_structured_log_opts_new ();
+
+   ASSERT_CMPINT (MONGOC_STRUCTURED_LOG_DEFAULT_MAX_DOCUMENT_LENGTH, ==, 1000);
+
+   ASSERT (mongoc_structured_log_opts_set_max_document_length (opts, 0));
+   ASSERT (!mongoc_structured_log_opts_set_max_document_length (opts, INT_MAX));
+   ASSERT (mongoc_structured_log_opts_set_max_document_length (opts, INT_MAX / 2));
+   ASSERT_CMPINT (INT_MAX / 2, ==, mongoc_structured_log_opts_get_max_document_length (opts));
+
+   mongoc_structured_log_opts_destroy (opts);
+}
+
+int
+check_test_structured_log_env_defaults (void)
```

Comment:
```suggestion
test_structured_log_skip_if_env_not_default (void)
```
Recommend including "skip" in the name of test suite skip functions for consistency.